Control/Submitter/src/ArmoniK.Core.Control.Submitter.csproj

```diff
@@ -48,4 +48,8 @@
     <ProjectReference Include="..\..\..\Common\src\ArmoniK.Core.Common.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+	<AssemblyAttribute Include="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute" />
```

Comment:
Why is this (non-test) project is excluded from coverage?

Reply:
This project contains the entry point for a submitter, main methods should not be tested but the methods it calls. Hence, I think they should be excluded from coverage.